.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,51 @@
+name: Lint and Test Charts
+
+on:
+  push:
+    branches-ignore:
+    - 'main'
+    - 'repository'
+    
+  pull_request:
+    branches:
+    - '**'
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.1
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Install chart-testing
+        uses: helm/chart-testing-action@v2.1.0
+
+#      - name: Run chart-testing (list-changed)
+#        id: list-changed
+#        run: |
+#          changed=$(ct list-changed --config ct.yaml)
+#          if [[ -n "$changed" ]]; then
+#            echo "::set-output name=changed::true"
+#          fi
+
+      - name: Run chart-testing (lint)
+        run: ct lint --config ct.yaml
+
+#      - name: Create kind cluster
+#        uses: helm/kind-action@v1.2.0
+#        if: steps.list-changed.outputs.changed == 'true'
+
+#      # See https://github.com/helm/chart-testing/blob/main/doc/ct_install.md
+#      - name: Run chart-testing (install)
+#        run: ct install --config ct.yaml
```

Comment:
This missing kind of defeats a core part of CI testing on the chart. 

What's the reason for this being removed?

Reply:
I included a reply to this comment above.